Actions/AL-Go-Helper.ps1

```diff
@@ -645,6 +645,75 @@ function ExcludeUnneededApps {
     }
 }
 
+function ResolveProjectFolders {
+    Param(
+        [string] $project,
+        [string] $baseFolder,
+        [ref] $projectSettings
+    )
+    $projectPath = Join-Path $baseFolder $project -Resolve
+
+    Push-Location $projectPath
+
+    try {
```

Comment:
What happens if we fail to resolve project folders? I guess this function would still succeed but would the workflow calling this function also succeed?

Reply:
The `try` is without a `catch` so a potential failure would fail the action.

But the function is generally "error free" (does not throw, given the assumption that the project folder exist, etc.). If an app folder path does not exist or there's no app.json, the path is simply ignored. There's a test to cover that scenario.